SocialNetwork/SocialNetwork.Core/Models/User.cs

```diff
@@ -3,4 +3,13 @@
 public class User
 {
     public int Id { get; set; }
+    public string Nickname { get; set; }
+    public string? Password { get; set; }
```

Comment:
In what scenario password can be null?

Reply:
Password may not be essential for creating an account